### PR TITLE
feat: healthcert 1.0

### PR DESCRIPTION
**Pull request by Nebulis**

### CODE REVIEW

**Comment by eujinong123** [comment]
I'd like to change `schemata/src/sg/gov/moh/healthcert/1.0/next-id-schema.json` with the version fully tested via OA signing.
[nextid-schema.txt](https://github.com/Open-Attestation/schemata/files/5442323/nextid-schema.txt)

IMO, the healthcert should have two orgs at least - one is a clinic, that is, healthcert provider and the other one is a lab. So the `minItems` should be 5 and don't have to mention `maxItems`, I think.

**Comment by Nebulis** [comment]
> I'd like to change `schemata/src/sg/gov/moh/healthcert/1.0/next-id-schema.json` with the version fully tested via OA signing [nextid-schema.txt](https://github.com/Open-Attestation/schemata/files/5442323/nextid-schema.txt).

No, I explained partially the reason by mail. On the other side I dont know if it's the first or second version that you sent but the first version was not event compatible with the sample provided.

> IMO, the healthcert should have two orgs at least - one is a clinic, that is, healthcert provider and the other one is a lab. So the minItems should be 5 and don't have to mention maxItems, I think.

I will probably change this to follow the fhir logic ... if there is any logic. At first look the JSON schema that exists don't check for the array size

**Comment by eujinong123** [comment]
> I will probably change this to follow the fhir logic ... if there is any logic.

In fact, FHIR logic doesn't insist it but I think we should consider the fact that the current healthcert was exclusively designed for proving a negative Covid-19 PCR test.

**Comment by Nebulis** [comment]
That's a good point but I also wonder if we should put the limitation here

**Comment by eujinong123** [comment]
[Clinic memo (22 Sep)-All countries .docx](https://github.com/Open-Attestation/schemata/files/5447704/Clinic.memo.22.Sep.-All.countries.docx)
This is the memo which MOH provided. According to the statements on it, the name of the lab should be mentioned.

**Comment by Nebulis** [comment]
@eujinong123 I've updated the schema a bit (check the commit for more details)

File: src/sg/gov/moh/healthcert/1.0/schema.json [hunk 1]
```diff
@@ -313,18 +343,22 @@
...
             "address": {
+              "description": "Visiting or postal addresses for the contact.",
               "type": "object",
               "required": ["type", "use", "text"],
               "properties": {
                 "type": {
-                  "type": "string",
+                  "description": "Distinguishes between physical addresses (those you can visit) and mailing addresses (e.g. PO Boxes and care-of addresses). Most addresses are both.",
+                  "enum": ["postal", "physical", "both"],
                   "examples": ["work"]
```
Comment (eujinong123) [inline src/sg/gov/moh/healthcert/1.0/schema.json:353]: This example value also should be updated, shouldn't it. Or eliminated.

File: src/sg/gov/moh/healthcert/1.0/schema.json [hunk 2]
```diff
@@ -313,18 +343,22 @@
...
                 },
                 "use": {
-                  "type": "string",
+                  "description": "The purpose of this address.",
+                  "enum": ["home", "work", "temp", "old", "billing"],
                   "examples": ["physical"]
```
Comment (eujinong123) [inline src/sg/gov/moh/healthcert/1.0/schema.json:358]: This example value also should be updated, shouldn't it. Or eliminated.

File: src/sg/gov/moh/healthcert/1.0/schema.json (hunk 1 above)
Comment (Nebulis) [reply src/sg/gov/moh/healthcert/1.0/schema.json:353]: Indeed

Comment (Nebulis) [reply src/sg/gov/moh/healthcert/1.0/schema.json]: updated

File: src/sg/gov/moh/healthcert/1.0/schema.json (hunk 2 above)
Comment (Nebulis) [reply src/sg/gov/moh/healthcert/1.0/schema.json]: updated

**Pull request merged**